simulator/server/di/di.go

```diff
@@ -10,37 +10,37 @@ import (
 	restclient "k8s.io/client-go/rest"
 	configv1 "k8s.io/kube-scheduler/config/v1"
 
-	"sigs.k8s.io/kube-scheduler-simulator/simulator/export"
+	"sigs.k8s.io/kube-scheduler-simulator/simulator/clusterresourceimporter"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/node"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/persistentvolume"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/persistentvolumeclaim"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/pod"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/priorityclass"
-	"sigs.k8s.io/kube-scheduler-simulator/simulator/replicateexistingcluster"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/reset"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/resourcewatcher"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/scheduler"
+	"sigs.k8s.io/kube-scheduler-simulator/simulator/snapshot"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/storageclass"
 )
 
 // Container saves and provides dependencies.
 type Container struct {
-	nodeService                     NodeService
-	podService                      PodService
-	pvService                       PersistentVolumeService
-	pvcService                      PersistentVolumeClaimService
-	storageClassService             StorageClassService
-	schedulerService                SchedulerService
-	exportService                   ExportService
-	priorityClassService            PriorityClassService
-	resetService                    ResetService
-	replicateExistingClusterService ReplicateExistingClusterService
-	resourceWatcherService          ResourceWatcherService
+	nodeService                  NodeService
+	podService                   PodService
+	pvService                    PersistentVolumeService
+	pvcService                   PersistentVolumeClaimService
+	storageClassService          StorageClassService
+	schedulerService             SchedulerService
+	snapshotService              SnapshotService
+	priorityClassService         PriorityClassService
+	resetService                 ResetService
+	importClusterResourceService ImportClusterResourceService
+	resourceWatcherService       ResourceWatcherService
 }
 
 // NewDIContainer initializes Container.
 // It initializes all service and puts to Container.
-// If externalImportEnabled is false, the simulator will not use externalClient and will not create ReplicateExistingClusterService.
+// If externalImportEnabled is false, the simulator will not use externalClient and will not create ImportClusterResourceService.
```

Comment:
to be easier to read.

```suggestion
// Only when externalImportEnabled is true, the simulator uses externalClient and creates ImportClusterResourceService.
```